assets/maps/dungeon_test

```diff
@@ -0,0 +1,6 @@
+height 20
```

Comment:
This data format seems alright, but it would be nice if this was like JSON for example.  I'm okay with this since we'll only have one dungeon (thus reading from only one file).